README.md

```diff
@@ -1,2 +1,187 @@
-# js-signer
- This repository provides library to sign any arbitrary message with an EOSIO R1 key and produce an EOSIO signature
+# NodeJS/Javascript library for Signing Messages
+
+This repository provides library to sign any arbitrary message with an EOSIO R1 key and produce an EOSIO signature
+
+## How to use in NodeJs
+
+### Online links for NodeJs
+
+- [Code / Playground Link 🔗](https://stackblitz.com/edit/node-js-signer?file=index.js)
+
+```js
+// run `node index.js` in the terminal
+
+// import the library
+const getSignature = require('js-signer').getSignature
+
+// import public private keys
+// for demo: keys are hardcoded here
+const privateKeyString = 'PVT_R1_ENSnpAGb4NHNA2chipxHQMVnAZdEAfRzHmJFEuxFkWvCXC5CG'
+const publicKeyString = 'PUB_R1_8CHJquaQWe4Pkhp1fBR9deP5wkqfjuWdfhaKYDxGKCo7gQwU9C'
+
+// create a function to generate singature for payload
+// this functions internally calls getSignature from library
+const signMessage = async message => await getSignature(message, publicKeyString, privateKeyString)
+
+// call the function with payload to generate signature
+const sigature = signMessage({ message: 'A very secret message' })
```

Comment:
This fails: https://github.com/bullish-exchange/js-signer/pull/1/files#r1039187589
because there is a typo here: you missed an `n`

Reply:
thanks